cluster/ubuntu/util.sh

```diff
@@ -450,7 +451,8 @@ function provision-node() {
       '${KUBELET_CONFIG}'
     create-kube-proxy-opts \
       '${1#*@}' \
-      '${MASTER_IP}' 
+      '${MASTER_IP}' \
+      '${KUBE_PROXY_EXTRA_OPTS}'
```

Comment:
should be ok since we're just using it to template a string, not part of an invocation